mock/py/mockbuild/util.py

```diff
@@ -476,7 +477,10 @@ def logOutput(fds, logger, returnOutput=1, start=0, timeout=0, printOutput=False
                     for line in lines:
                         if line != '':
                             line = ansi_escape.sub('', line)
-                            logger.debug(line)
+                            if fderr is s and not line.startswith('+ '):
+                                logger.debug("BUILDSTDERR: " + line)
```

Comment:
Can this be rather: `logger.debug("BUILDSTDERR: %s", line)`?

See https://www.simonmweber.com/2014/11/24/python-logging-traps.html